enclave_wrangler/main.py

```diff
@@ -46,19 +49,43 @@ def _datetime_palantir_format() -> str:
     return datetime.now(timezone.utc).strftime("%Y-%m-%dT%H:%M:%S.%fZ")[:-4] + 'Z'
 
 
-def run(input_csv_folder_path):
-    """Main function"""
+def _load_standardized_input_df(
```

Comment:
This standardized way of loading our known input dataframes is now being re-used 3 times in the `enclave_wrangler`, so it seemed worthwhile to do this.